app/build.gradle.kts

```diff
@@ -25,7 +25,21 @@ android {
         }
     }
 
+    signingConfigs {
+        create("release") {
+            // You need to specify either an absolute path or include the
+            // keystore file in the same directory as the build.gradle file.
+            storeFile = file("EndurAi.jks")
+            storePassword = "EndurAi"
+            keyAlias = "EndurAiKeyStore"
+            keyPassword = "EndurAi"
```

Comment:
Maybe it would be better if we add it to a repository secret

But There is already one secret key, so i guess it's fine.